scrapy/linkextractors/__init__.py

```diff
@@ -15,7 +15,7 @@
 from scrapy.utils.url import (
     url_is_from_any_domain, url_has_any_extension,
 )
-
+from scrapy.utils.python import unique as unique_list
```

Comment:
:lipstick: Could you move it into the group of import above, keeping the alphabetical sorting?